src/utils.tsx

```diff
@@ -16,7 +15,7 @@ export const typeToUnit = { Pixel: "px", Percentage: "%", Ephemeral: "em" };
 export function mjmlReactComponentFactory<P>(
```

Comment:
I think we should actually delete this function as it is unused, but doesn't need to be done in this PR

Reply:
will do that here